src/functions/getJournal/framework/handler.ts

```diff
@@ -13,7 +12,7 @@ export async function handler(event: APIGatewayProxyEvent, fnCtx: Context) {
   }
 
   if (process.env.EMPLOYEE_ID_VERIFICATION_DISABLED !== 'true') {
-    const employeeId = getEmployeeIdFromToken(event.headers.Authorization);
+    const employeeId = getEmployeeIdFromRequestContext(event.requestContext);
     if (employeeId === null) {
       return createResponse('Invalid authorisation token', HttpStatus.UNAUTHORIZED);
```

Comment:
Should probably update the error too as no longer using token for authentication

Reply:
Good shot @roberthall12, done.